src/sentry/users/api/endpoints/user_details.py

```diff
@@ -38,6 +39,17 @@
 TIMEZONE_CHOICES = get_timezone_choices()
 
 
+def validate_quick_start_display(value: dict[str, int] | None) -> None:
+    if value is not None:
+        for display_value in value.values():
```

Comment:
Are there any constraints/rules for the keys?

Reply:
Ideally, the key should be the active organization, but since the endpoint doesn’t provide it, shall we retrieve it from the session?